sunkit_instruments/iris/tests/test_iris_tools.py

```diff
@@ -1,308 +1,320 @@
-# -*- coding: utf-8 -*-
```

Comment:
What is gonna happen to this file?

Reply:
@nabobalis Dan instructed me to comment this file out as a giant docstring for now